### PR TITLE
Improve the UI capability of CMEK

### PR DESCRIPTION
**Pull request by ljun0712**
### What is changed, added or deleted? (Required)

Because the CMEK feature provides UI capabilities, we need to update the feature documentation.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 1]
```diff
@@ -24,82 +24,71 @@ After you enable CMEK for a project, all clusters created within this project en
...
+6. Fill in the Key ARN obtained from AWS KMS.
+7. Click **Create** to complete the creation of the key.
 
-2. Configure the KMS Key ARN for the specified region (for example, us-east-1) under this project using the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint of TiDB Cloud API.
+</div>
+<div label="Use API" value="api">
+
+You can complete this step using TiDB Cloud API through the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint.
+
+Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:79]: 建议把 Appendix 里的内容放到这里，放结尾容易看漏信息：

```suggestion
1. Configure the key policy on AWS KMS, and add the following information to the key policy:

    ```json
    {
        "Version": "2012-10-17",
        "Id": "cmek-policy",
        "Statement": [
            // EBS-related policy
            {
                "Sid": "Allow access through EBS for all principals in the account that are authorized to use EBS",
                "Effect": "Allow",
                "Principal": {
                    "AWS": "*"
                },
                "Action": [
                    "kms:Encrypt",
                    "kms:Decrypt",
                    "kms:ReEncrypt*",
                    "kms:GenerateDataKey*",
                    "kms:CreateGrant",
                    "kms:DescribeKey"
                ],
                "Resource": "*",
                "Condition": {
                    "StringEquals": {
                        "kms:CallerAccount": "<pingcap-account>",
                        "kms:ViaService": "ec2.<region>.amazonaws.com"
                    }
                }
            },
            // S3-related policy
            {
                "Sid": "Allow TiDB cloud role to use KMS to store encrypted backup to S3",
                "Effect": "Allow",
                "Principal": {
                    "AWS": "arn:aws:iam::<pingcap-account>:root"
                },
                "Action": [
                    "kms:Decrypt",
                    "kms:GenerateDataKey"
                ],
                "Resource": "*"
            },
            ... // user's own admin access to KMS
        ]
    }
    ```

    - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
    - `<region>` is the region where you want to create your cluster, for example, `us-west-2`. If you do not want to specify a region, replace `<region>` with a wildcard `*`, and put it in a `StringLike` block.
    - For EBS-related policy in the preceding block, refer to [AWS documentation](https://docs.aws.amazon.com/kms/latest/developerguide/conditions-kms.html#conditions-kms-caller-account).
    - For S3-related policy in the preceding block, refer to [AWS blog](https://repost.aws/knowledge-center/s3-bucket-access-default-encryption).

2. Call the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint of TiDB Cloud API.

    Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 2]
```diff
@@ -24,82 +24,71 @@ After you enable CMEK for a project, all clusters created within this project en
...
+
+</div>
+</SimpleTab>
+
+> **Note:**
+>
+> - This feature is in the development stage and the policy requirement could be subject to change as future features might require more permissions. 
+> - For projects that do not have CMEK enabled, you can view the Encryption Access page, but you are not allowed to click Create New Key.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:87]:
```suggestion
> - This feature is in the development stage and the policy requirement is subject to change as future features might require more permissions. 
> - For projects without CMEK enabled, you can access the **Encryption Access** page but are not allowed to create a new key.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 3]
```diff
@@ -124,3 +113,60 @@ After revoking TiDB Cloud's access to CMEK, if you need to restore the access, f
...
+    ]
+}
+```
+
+> **Note:**
+>
+> - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
+> - `<region>` is the region where you want to create your cluster, for example, `us-west-2`. If you do not want to specify a region, replace `<region>` with a wildcard `*`, and put it in a `StringLike` block.
+> - For EBS-related policy in the preceding block, refer to [AWS documentation](https://docs.aws.amazon.com/kms/latest/developerguide/conditions-kms.html#conditions-kms-caller-account).
+> - For S3-related policy in the preceding block, refer to [AWS blog](https://repost.aws/knowledge-center/s3-bucket-access-default-encryption).
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:172]: 移到上面 API 部分
```suggestion
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 4]
```diff
@@ -24,82 +24,71 @@ After you enable CMEK for a project, all clusters created within this project en
...
-### Step 2. Create a new project and enable CMEK
+<SimpleTab groupId="method">
+<div label="Use Console" value="console">
+
+You need to create a CMEK-enabled project, take the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
+2. Click **Organization Settings**. You can enter your organization management page.
+3. Click **Create New Project** to enter the project creation page.
+4. Fill in a project name
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:39]:
```suggestion
4. Fill in a project name.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 5]
```diff
@@ -24,82 +24,71 @@ After you enable CMEK for a project, all clusters created within this project en
...
+You can complete the CMEK configuration of the project either using the console or using API.
+
+<SimpleTab groupId="method">
+<div label="Use Console" value="console">
 
-1. Create a new project and enable AWS CMEK using the [Create a project](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Project/operation/CreateProject) endpoint of TiDB Cloud API.
+To complete the CMEK configuration of the project, take the following steps:
 
-    Make sure the `aws_cmek_enabled` field is set to `true`.
+1.  Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:66]:
```suggestion
1. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 6]
```diff
@@ -24,82 +24,71 @@ After you enable CMEK for a project, all clusters created within this project en
...
+2. Click **Encryption Access** to enter the encryption management page of the project.
+3. Click **Create Encryption Key** to enter the key creation page.
+4. The key provider only supports AWS KMS. You can choose the region where the encryption key is used.
+5. Copy and save JSON file as ROLE-TRUST-POLICY.JSON，This file describes the trust relationship. Please add this trust relationship to the key policy of AWS KMS. Please refer to [Key policies in AWS KMS](https://docs.aws.amazon.com/zh_cn/kms/latest/developerguide/key-policies.html).
+6. Fill in the Key ARN obtained from AWS KMS.
+7. Click **Create** to complete the creation of the key.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:72]:
```suggestion
5. Copy and save the JSON file as `ROLE-TRUST-POLICY.JSON`. This file describes the trust relationship. 
6. Add this trust relationship to the key policy of AWS KMS. For more information, refer to [Key policies in AWS KMS](https://docs.aws.amazon.com/zh_cn/kms/latest/developerguide/key-policies.html).
7. Fill in the Key ARN obtained from AWS KMS.
8. Click **Create** to finish creating the key.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md
[the remark this answers is not in the document]
Comment (ljun0712) [reply tidb-cloud/tidb-cloud-encrypt-cmek.md:1]: down

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 7]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
 
-1. Create your CMEK Key on the AWS Key Management Service (KMS) console. Copy the KMS Key ARN. To learn how to create a key, see [Creating Keys](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk) in the AWS documentation.
+You can create a CMEK-enabled project either using the console or using API.
 
-2. Create an IAM role and configure the role's access policy to CMEK.
+<SimpleTab groupId="method">
+<div label="Use Console" value="console">
 
-    To make the TiDB cluster function as expected, you need to grant certain permissions to the TiDB account for accessing KMS. Note that this feature is in the development stage and the policy requirement could be subject to change as future features might require more permissions. Here’re the required permissions for supported features at present:
+You need to create a CMEK-enabled project, take the following steps:
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:34]:
```suggestion
To create a CMEK-enabled project, take the following steps:
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 8]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
+2. Click **Organization Settings**. You can enter your organization management page.
+3. Click **Create New Project** to enter the project creation page.
+4. Fill in a project name.
+5. Choose to enable the CMEK capability of the Project.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:40]:
```suggestion
5. Choose to enable the CMEK capability of the project.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:1]: Preview environment: https://preview-cloud-pingcap-docs-1.pingcap-docs-website-scaffold-1hq.pages.dev/tidbcloud/tidb-cloud-encrypt-cmek

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 9]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+</div>
+</SimpleTab>
+
+### Step 2. Complete the CMEK configuration of the project
+
+You can complete the CMEK configuration of the project either using the console or using API.
+
+> **Note:**
+>
+> You must ensure that the policy of the key meets the requirements, such as: insufficient permissions, account errors, etc. These errors will cause the cluster to be incorrectly created using this key.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:63]: 这里的意思是“确保密钥的策略符合要求，并且没有权限不足、账户错误等错误“吗
```suggestion
> You must ensure that the policy of the key meets the requirements and is free from errors such as insufficient permissions or account issues. These errors will cause the cluster to be incorrectly created using this key.
```

File: TOC-tidb-cloud.md [hunk 10]
```diff
@@ -228,6 +228,8 @@
       - [Connect via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md)
       - [Connect via VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md)
       - [TLS Connections to TiDB Dedicated](/tidb-cloud/tidb-cloud-tls-connect-to-dedicated.md)
+  - Data Access Control
+      - [Encryption at Rest Using Customer-Managed Encryption Keys](tidb-cloud/tidb-cloud-encrypt-cmek.md)
```
Comment (ran-huang) [inline TOC-tidb-cloud.md:232]:
```suggestion
      - [Encryption at Rest Using Customer-Managed Encryption Keys](/tidb-cloud/tidb-cloud-encrypt-cmek.md)
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 11]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
 
 If you want to encrypt your data using the KMS owned by your account, take the following steps.
 
-### Step 1. Provision KMS and IAM in your cloud provider
+### Step 1. Create a CMEK-enabled project
 
-1. Create your CMEK Key on the AWS Key Management Service (KMS) console. Copy the KMS Key ARN. To learn how to create a key, see [Creating Keys](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk) in the AWS documentation.
+You can create a CMEK-enabled project either using the console or using API.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:29]:
```suggestion
If you are in the `Organization Owner` role of your organization, you can create a CMEK-enabled project using either the TiDB Cloud console or API.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 12]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
 
-2. Create an IAM role and configure the role's access policy to CMEK.
+<SimpleTab groupId="method">
+<div label="Use Console" value="console">
 
-    To make the TiDB cluster function as expected, you need to grant certain permissions to the TiDB account for accessing KMS. Note that this feature is in the development stage and the policy requirement could be subject to change as future features might require more permissions. Here’re the required permissions for supported features at present:
+To create a CMEK-enabled project, take the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
+2. Click **Organization Settings**. You can enter your organization management page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:37]:
```suggestion
2. Click **Organization Settings**.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 13]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+3. Click **Create New Project** to enter the project creation page.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:38]:
```suggestion
3. On the **Organization Settings** page, click **Create New Project** to open the project creation dialog.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 14]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+You can complete this step using TiDB Cloud API through the [Create a CMEK-enabled project](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Project/operation/CreateProject) endpoint. Make sure that the `aws_cmek_enabled` field is set to `true`.
+
+Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
+
+</div>
+</SimpleTab>
+
+### Step 2. Complete the CMEK configuration of the project
+
+You can complete the CMEK configuration of the project either using the console or using API.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:59]:
```suggestion
You can complete the CMEK configuration of the project using either the TiDB Cloud console or API.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 15]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+> **Note:**
+>
+> You must ensure that the policy of the key meets the requirements, such as: insufficient permissions, account errors, etc. These errors will cause the cluster to be incorrectly created using this key.
+
+<SimpleTab groupId="method">
+<div label="Use Console" value="console">
+
+To complete the CMEK configuration of the project, take the following steps:
+
+1. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Admin**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:70]:
```suggestion
1. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Project Settings**.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 16]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+2. Click **Encryption Access** to enter the encryption management page of the project.
+3. Click **Create Encryption Key** to enter the key creation page.
+4. The key provider only supports AWS KMS. You can choose the region where the encryption key is used.
+5. Copy and save the JSON file as `ROLE-TRUST-POLICY.JSON`. This file describes the trust relationship. 
+6. Add this trust relationship to the key policy of AWS KMS. For more information, refer to [Key policies in AWS KMS](https://docs.aws.amazon.com/zh_cn/kms/latest/developerguide/key-policies.html).
+7. Fill in the Key ARN obtained from AWS KMS.
+8. Click **Create** to finish creating the key.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:77]:
```suggestion
8. Click **Create** to create the key.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 17]
```diff
@@ -78,28 +127,26 @@ If you want to encrypt your data using the KMS owned by your account, take the f
...
-2. Configure the KMS Key ARN for the specified region (for example, us-east-1) under this project using the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint of TiDB Cloud API.
+> **Note:**
+>
+> - This feature is in the development stage and the policy requirement is subject to change as future features might require more permissions. 
+> - For projects without CMEK enabled, you can access the **Encryption Access** page but are not allowed to create a new key.
 
 ### Step 3. Create a cluster
 
-Create a TiDB Dedicated cluster hosted on AWS under the project created in Step 1. Ensure that the region where the cluster is located is the same as that in Step 2.
+Under the project created in Step 1, create a TiDB Dedicated cluster hosted on AWS. For detailed steps, refer to [this document](https://docs.pingcap.com/tidbcloud/create-tidb-cluster). Ensure that the region where the cluster is located is the same as that in Step 2.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:149]:
```suggestion
Under the project created in [Step 1](#step-1-create-a-cmek-enabled-project), create a TiDB Dedicated cluster hosted on AWS. For detailed steps, refer to [this document](/tidb-cloud/create-tidb-cluster.md). Ensure that the region where the cluster is located is the same as that in [Step 2](/tidb-cloud/tidb-cloud-encrypt-cmek.md#step-2-complete-the-cmek-configuration-of-the-project).
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 18]
```diff
@@ -78,28 +127,26 @@ If you want to encrypt your data using the KMS owned by your account, take the f
...
 
-> **Note:**
->
-> - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
-> - `<region>` is the region where you want to create your cluster, for example, `us-west-2`. If you do not want to specify a region, replace `<region>` with a wildcard `*`, and put it in a `StringLike` block.
-> - For EBS-related policy in the preceding block, refer to [AWS documentation](https://docs.aws.amazon.com/kms/latest/developerguide/conditions-kms.html#conditions-kms-caller-account).
-> - For S3-related policy in the preceding block, refer to [AWS blog](https://repost.aws/knowledge-center/s3-bucket-access-default-encryption).
-
-### Step 2. Create a new project and enable CMEK
+    - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:130]:
```suggestion
    - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 19]
```diff
@@ -78,28 +127,26 @@ If you want to encrypt your data using the KMS owned by your account, take the f
...
+    Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
 
-    Make sure the `aws_cmek_enabled` field is set to `true`.
+</div>
+</SimpleTab>
 
-2. Configure the KMS Key ARN for the specified region (for example, us-east-1) under this project using the [Configure AWS CMEK](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/CreateAwsCmek) endpoint of TiDB Cloud API.
+> **Note:**
+>
+> - This feature is in the development stage and the policy requirement is subject to change as future features might require more permissions.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:144]: 现在已经 GA 了，这里需要调整下描述吗

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 20]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+
+To complete the CMEK configuration of the project, take the following steps:
+
+1. Click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner, switch to the target project if you have multiple projects, and then click **Project Settings**.
+2. Click **Encryption Access** to enter the encryption management page of the project.
+3. Click **Create Encryption Key** to enter the key creation page.
+4. The key provider only supports AWS KMS. You can choose the region where the encryption key is used.
+5. Copy and save the JSON file as `ROLE-TRUST-POLICY.JSON`. This file describes the trust relationship. 
+6. Add this trust relationship to the key policy of AWS KMS. For more information, refer to [Key policies in AWS KMS](https://docs.aws.amazon.com/zh_cn/kms/latest/developerguide/key-policies.html).
+7. Fill in the Key ARN obtained from AWS KMS.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:76]:
```suggestion
7. Fill in the **KMS Key ARN** obtained from AWS KMS.
```

File: tidb-cloud/tidb-cloud-encrypt-cmek.md [hunk 21]
```diff
@@ -24,13 +24,62 @@ After you enable CMEK for a project, all clusters created within this project en
...
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
+2. Click **Organization Settings**.
+3. On the **Organization Settings** page, click **Create New Project** to open the project creation dialog.
+4. Fill in a project name.
+5. Choose to enable the CMEK capability of the project.
+6. Click **Confirm** to complete the project creation.
+
+> **Note:**
+>
+> After creating a CMEK-enabled project, you will be redirected to the Encryption Access page so that you can complete the CMEK configuration.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-encrypt-cmek.md:45]: 点了 Confirm 之后，没有自动跳转到 Encryption Access page，是环境的问题吗

**Pull request merged**